docker/OWNERS

```diff
@@ -1,4 +0,0 @@
-approvers:
```

Comment:
why remove this? I think it's appropriate the users if the dockerfiles are the owners rather than prow team. Although long term I guess these will be in tools